src/material/schematics/ng-generate/mdc-migration/rules/components/slider/slider-template.ts

```diff
@@ -52,12 +52,28 @@ export class SliderTemplateMigrator extends TemplateMigrator {
             updates.push(this._removeBinding(originalHtml, binding.node));
           }
 
-          if (binding.name === 'invert' || binding.name === 'vertical') {
+          if (
+            binding.name === 'invert' ||
+            binding.name === 'vertical' ||
+            binding.name === 'tickInterval' ||
+            binding.name === 'valueText' ||
+            binding.name === 'defaultColor' ||
```

Comment:
`defaultColor`, `defaultTabIndex` and `onTouched` weren't inputs so we don't need to migrate them.

Reply:
I figured something weird was going on here. I assume they aren't meant to be public but they were listed on the docs site so I added them here (https://material.angular.io/components/slider/api). Should I still remove them?

Reply:
They show up there, because they're public properties of the slider, but they can't be bound to in a template since they aren't inputs.